script/dev-deploy

```diff
@@ -86,12 +86,12 @@ def is_container_running(remote, container_name, container_bin="docker"):
     return False
 
 
-def scp_command(local_file, remote, remote_file) -> str:
+def scp_command(local_file, remote, remote_file, extra_opts="") -> str:
     cmd = ""
     file_dir = os.path.dirname(remote_file)
     if file_dir != "":
         cmd += f"ssh {remote} 'mkdir -p {file_dir}' && "
-    cmd += f"scp {local_file} {remote}:{remote_file}"
+    cmd += f"scp {extra_opts} {local_file} {remote}:{remote_file}"
```

Comment:
use `cmd += f"scp -r {local_file} {remote}:{remote_file}"` directly, its valid for both file and directory transfers

Reply:
ok, thx